### PR TITLE
sample json

### PR DESCRIPTION
**Pull request by ticoann**
add the sample json example. @vkuznet, @yuyiguo  please review. And feel free to delete the wiki document

### CODE REVIEW

File: test/data/samples/fwjr_production.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+                         'size': 647376,
+                         #TODO change to empty string 
+                         'user_dn': '',
+                         'user_vogroup': 'DEFAULT',
+                         'user_vorole': 'DEFAULT',
+                         'validStatus': 'PRODUCTION'}],
+              'performance': {#"storage": {},
+                   #"multicore": {},
+                   "memory": {
+                       "PeakValueRss": "0",
```
Comment (vkuznet) [inline test/data/samples/fwjr_production.py]: Why RSS and Vsize values are strings?

Comment (ticoann) [inline test/data/samples/fwjr_production.py]: Sorry my mistake, updated now

**Comment by vkuznet** [comment]
@yuyiguo I don't think we need to apply formatter to json files. They're in json format. If you need to read it, just load it in your python shell like that

```
import pprint, json
rec = json.load(open("fwjr.json"))
pprint.pprint(rec)
```

and you'll get nice formatted output.

**Comment by yuyiguo** [comment]
@vkuznet It is not problem to format in the shell. It am talking about browsing the schema in github. I realized that the *py files are formated.

**Comment by ticoann** [comment]
Hi Yuyi, json files are not for the browsing they are for the feed for creating schema.
That is why I added 2 additional python module which generates those two json files.  
So you can only focus on 2 python modules instead of json files for browsing.

As Valentin pointed you can use pprint to get the formatted data but that is exactly what python module has in this pull request.

**Pull request merged**